backend/src/test/java/com/redhat/cloud/notifications/processors/email/EmailTest.java

```diff
@@ -157,47 +147,41 @@ void testEmailSubscriptionInstant() {
             mockServerConfig.getMockServerClient().clear(postReq);
         }
 
-        assertEquals(1, bodyRequests.size());
-        JsonObject body = new JsonObject(bodyRequests.get(0));
-        JsonArray emails = body.getJsonArray("emails");
-        assertNotNull(emails);
-        assertEquals(1, emails.size());
-        JsonObject firstEmail = emails.getJsonObject(0);
-        JsonArray recipients = firstEmail.getJsonArray("recipients");
-        assertEquals(1, recipients.size());
-        assertEquals("no-reply@redhat.com", recipients.getString(0));
-
-        JsonArray bccList = firstEmail.getJsonArray("bccList");
-        assertEquals(usernames.length, bccList.size());
-
-        List<String> sortedUsernames = Arrays.asList(usernames);
-        sortedUsernames.sort(Comparator.naturalOrder());
-
-        List<String> sortedBccList = new ArrayList<String>(bccList.getList());
-        sortedBccList.sort(Comparator.naturalOrder());
+        assertEquals(3, bodyRequests.size());
+        for (int i = 0; i < usernames.length; ++i) {
+            JsonObject body = new JsonObject(bodyRequests.get(i));
+            JsonArray emails = body.getJsonArray("emails");
+            assertNotNull(emails);
+            assertEquals(1, emails.size());
+            JsonObject firstEmail = emails.getJsonObject(0);
+            JsonArray recipients = firstEmail.getJsonArray("recipients");
+            assertEquals(1, recipients.size());
+            assertEquals(usernames[i], recipients.getString(0));
```

Comment:
Emails could arrive in any order. I need to ensure these are sorted (as done elsewhere) before comparing